README.md

```diff
@@ -37,7 +37,7 @@ export default class MainScreenComponent extends Component {
       icon: Image.resolveAssetSource(exitIcon).uri,
       id: 'exit',
       location: 'right',
-      accessibilityLabel: 'Exit this app'
+      adaLabel: 'Exit this app'
```

Comment:
Best to update this in `src/components/navigation-component.js` as well (and regenerate the docs: `yarn docs`), so that the website will be updated as well.